airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/formatter/DefaultBigQueryRecordFormatter.java

```diff
@@ -34,26 +34,21 @@ public class DefaultBigQueryRecordFormatter extends BigQueryRecordFormatter {
       Field.of(JavaBaseConstants.COLUMN_NAME_AB_RAW_ID, StandardSQLTypeName.STRING),
       Field.of(JavaBaseConstants.COLUMN_NAME_AB_EXTRACTED_AT, StandardSQLTypeName.TIMESTAMP),
       Field.of(JavaBaseConstants.COLUMN_NAME_AB_LOADED_AT, StandardSQLTypeName.TIMESTAMP),
-      Field.of(JavaBaseConstants.COLUMN_NAME_DATA, StandardSQLTypeName.JSON));
+      Field.of(JavaBaseConstants.COLUMN_NAME_DATA, StandardSQLTypeName.STRING));
 
-  public DefaultBigQueryRecordFormatter(JsonNode jsonSchema, StandardNameTransformer namingResolver) {
+  public DefaultBigQueryRecordFormatter(final JsonNode jsonSchema, final StandardNameTransformer namingResolver) {
     super(jsonSchema, namingResolver);
   }
 
   @Override
-  public JsonNode formatRecord(AirbyteRecordMessage recordMessage) {
+  public JsonNode formatRecord(final AirbyteRecordMessage recordMessage) {
     if (TypingAndDedupingFlag.isDestinationV2()) {
       // Map.of has a @NonNull requirement, so creating a new Hash map
       final HashMap<String, Object> destinationV2record = new HashMap<>();
       destinationV2record.put(JavaBaseConstants.COLUMN_NAME_AB_RAW_ID, UUID.randomUUID().toString());
       destinationV2record.put(JavaBaseConstants.COLUMN_NAME_AB_EXTRACTED_AT, getEmittedAtField(recordMessage));
       destinationV2record.put(JavaBaseConstants.COLUMN_NAME_AB_LOADED_AT, null);
-      if (useObjectForData()) {
-        destinationV2record.put(JavaBaseConstants.COLUMN_NAME_DATA,
-            StandardNameTransformer.formatJsonPath(recordMessage.getData()));
-      } else {
-        destinationV2record.put(JavaBaseConstants.COLUMN_NAME_DATA, getData(recordMessage));
-      }
+      destinationV2record.put(JavaBaseConstants.COLUMN_NAME_DATA, getData(recordMessage));
```

Comment:
@tryangul fyi, standard inserts mode is modifying how `_airbyte_data` gets formatted (in addition to changing the column type json -> string on line 37). Hopefully nothing will break from your perspective 🤞